packages/dashboard/src/components/tasks/task-page.tsx

```diff
@@ -92,23 +92,45 @@ export function TaskPage() {
   );
 
   return (
-    <TaskPanel
-      className={classes.taskPanel}
-      tasks={autoRefreshState.tasks}
-      paginationOptions={{
-        page,
-        count: hasMore ? -1 : page * 10 + autoRefreshState.tasks.length,
-        rowsPerPage: 10,
-        rowsPerPageOptions: [10],
-        onChangePage: (_ev, newPage) => setPage(newPage),
-      }}
-      cleaningZones={placeNames}
-      loopWaypoints={placeNames}
-      deliveryWaypoints={placeNames}
-      submitTasks={submitTasks}
-      cancelTask={cancelTask}
-      onRefresh={handleRefresh}
-      onAutoRefresh={autoRefreshDispatcher.setEnabled}
-    />
+    <div>
+      <TaskPanel
```

Comment:
I assume the 2 task panels is just for demo?

Reply:
yes this is just for demo purposes, will remove when we've made a decision on which alternative to use